test/fixtures/output/service-quota/1.golden

```diff
@@ -1,6 +1,6 @@
   Quota Code |  Name   |     Scope     | Applied Limit | Usage | Organization | Environment | Kafka Cluster | Network | User  
 -------------+---------+---------------+---------------+-------+--------------+-------------+---------------+---------+-------
-  quota_a    | Quota A | kafka_cluster |            15 |    10 | org-123      | env-1       | lkc-1         |         |       
-  quota_a    | Qutoa A | kafka_cluster |            16 |    11 | org-123      | env-2       | lkc-2         |         |       
+  quota_a    | Quota A | kafka_cluster |            15 | 10    | org-123      | env-1       | lkc-1         |         |       
```

Comment:
Reposting because I highlighted the wrong line last time.

`Applied Limit` is `int32` in the `quotaValue` struct, so shouldn't it also be left-aligned here? Same w/ `Partition`, `Partition Mirror Lag`, etc. in the mirror commands.

Reply:
Integers are intentionally right-aligned. "Usage" is a string so it is left-aligned (they wanted to be able to set it to `""` when the value was 0 since 0 would convey a different meaning here)

Reply:
Ohh, got it.